### PR TITLE
Lisa2

### PR DESCRIPTION
**Pull request by baigal628**
Integration of LISA2 into MAESTRO workflow:
Changes mainly in 
1. R package RNAAnnotateTranscriptionFactor() function.
2. Command line parsers for new LISA
3. Relevant documentations

### CODE REVIEW

File: MAESTRO/MAESTRO_PipeInit.py [hunk 1]
```diff
@@ -176,37 +176,37 @@ def scrna_parser(subparsers):
...
-        choices = ["GRCh38", "GRCm38"], type = str, 
+        choices = ["GRCh38", "GRCm38"], type = str,
         help = "Specify the genome assembly (GRCh38 for human and GRCm38 for mouse). DEFAULT: GRCh38.")
 
     # Output arguments
     group_output = workflow.add_argument_group("Running and output arguments")
-    group_output.add_argument("--cores", dest = "cores", default = 8, 
-        type = int, help = "The number of cores to use. DEFAULT: 8.")
-    group_output.add_argument("--rseqc", dest = "rseqc", action = "store_true", 
+    group_output.add_argument("--cores", dest = "cores", default = 10,
```
Comment (crazyhottommy) [inline MAESTRO/MAESTRO_PipeInit.py:201]: 10 cores is the best for lisa right? I remember Allen was saying

Comment (baigal628) [reply MAESTRO/MAESTRO_PipeInit.py:201]: Yes. So I changed the default to 10.

**Comment by crazyhottommy** [comment]
the check has passed. After merging, @taoliu the MAESTRO package will be uploaded to liulab conda channel right? or need to make a tag again?

**Comment by taoliu** [comment]
> the check has passed. After merging, @taoliu the MAESTRO package will be uploaded to liulab conda channel right? or need to make a tag again?

We need to 'tag' a release in master branch.

File: MAESTRO/scATAC_H5Process.py [hunk 2]
```diff
@@ -24,63 +24,63 @@ def mtxtoh5_parser(subparsers):
...
         "If the type is Peak, please provide the peak bed file with 3 columns. "
         "If the type is Gene, each row should include gene name. DEFAULT: features.tsv.")
     group_input.add_argument("--gene-column", dest = "gene_column", default = 2, type = int,
         help = "If the type is 'Peak', please specify which column of the feature file to use for gene names. DEFAULT: 2.")
-    group_input.add_argument("--barcode", dest = "barcode", default = "barcodes.tsv", 
+    group_input.add_argument("--barcode", dest = "barcode", default = "barcodes.tsv",
         help = "Location of barcode file. Cell barcodes correspond to column indices of count matrix. DEFAULT: barcodes.tsv. ")
-    group_input.add_argument("--species", dest = "species", default = "GRCh38", 
-        choices = ["GRCh38", "GRCm38"], type = str, 
+    group_input.add_argument("--species", dest = "species", default = "GRCh38",
```
Comment (taoliu) [inline MAESTRO/scATAC_H5Process.py:43]: I feel strange to see this argument as 'species' but actually referring to 'genome assembly versions'.

**Review by taoliu — APPROVED** [review]

**Comment by crazyhottommy** [comment]
Just realized that the README is not updated. @baigal628 you already have it.

**Comment by crazyhottommy** [comment]
It is there. sorry about that. All set. I had a release draft @taoliu  you can do a formal release if you want.

**Comment by crazyhottommy** [comment]
@taoliu what's wrong with Travis? the build after tag1.2.2 failed.

**Comment by crazyhottommy** [comment]
can you @taoliu trigger the re-build? I can not by myself..

**Pull request merged**